app/models/work.rb

```diff
@@ -1230,4 +1230,9 @@ def nonfiction
     nonfiction_tags = [125773, 66586, 123921, 747397] # Essays, Nonfiction, Reviews, Reference
     (filter_ids & nonfiction_tags).present?
   end
+
+  # If the creator(s) allow this work to be invited to collections.
+  def allow_collection_invitation?
+    users.all? { |user| user.preference.allow_collection_invitation }
```

Comment:
This likely needs an official opinion, but I noticed that the Jira issue says
> When none of the creators of a work has checked "Allow others to invite my works to collections.", the "Add To Collections" button should be removed from the work. 

Which I would read as "show if at least one person allows it", so in this case I think it should be `any?` instead of `all?`.

Reply:
Yeah, this should be `any?`. Good catch!

Reply:
Ok! I will update this. I think I went `all?` originall because it seemed odd to have one creator's opt-in override their co-creator's (or creators') opt-out -- or at least lack of opt-in -- but I guess that can be solved with good communication :+1:

Reply:
I think we went with `any?` to be consistent with how the `disable_share_links?` preference works: opting out only if everyone opts out.

https://github.com/otwcode/otwarchive/blob/2d92ec1e100d2db8d98c791258b8ea115044975d/app/views/works/_work_header_navigation.html.erb#L99-L103

We'll see if this (and/or the FAQ text) needs tweaking further.